openslides/users/static/js/users/site.js

```diff
@@ -445,10 +445,27 @@ angular.module('OpenSlidesApp.users.site', [
             }
         });
         $scope.alert = {};
-        $scope.groupFilter = undefined;
 
         // setup table sorting
-        $scope.sortColumn = 'first_name'; //TODO: sort by first OR last name
+        $scope.sortOptions = [
+            {name: 'first_name',
+             display_name: 'First name'},
+            {name: 'last_name',
+             display_name: 'Last name'},
+            {name: 'is_present',
+             display_name: 'Present'},
+            {name: 'is_active',
+             display_name: 'Active'},
+            {name: 'is_committee',
+             display_name: 'Committee'},
+            {name: 'number',
+             display_name: 'Number'},
+            {name: 'structure_level',
+             display_name: 'Structure level'},
+            {name: 'comment',
+             display_name: 'Comment'},
+        ];
+        $scope.sortColumn = ($scope.config('users_sort_by') == 'firstName') ? 'first_name' : 'last_name';
```

Comment:
It would be better if the config option uses underscore and not camel case, so we can better rely on the model attribute which is also underscore in JavaScript.